Tasks/AzureDevOpsTestPlanV0/manualTests.ts

```diff
@@ -0,0 +1,6 @@
+import tl = require('azure-pipelines-task-lib/task');
+
+export function manualTestsFlow() {
```

Comment:
Just wanted to check what functionality is this function doing? Since we are only having 2 console.log statements here.

Reply:
This is a to do scenario, where we will be adding the functionality of manual tests